packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillImageStyles.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillImageStylesProps } from '../../../../components/Pill/PillImage';
+import { PillVariables } from './pillVariables';
+
+export const pillImageStyles: ComponentSlotStylesPrepared<PillImageStylesProps, PillVariables> = {
```

Comment:
We need to make sure to keep exporting the style props interfaces, otherwise consumers have no way of knowing what props are available for overrides

Reply:
Done